vowpalwabbit/interactions.cc

```diff
@@ -239,7 +239,7 @@ void eval_count_of_generated_ft(vw& all, example& ec, size_t& new_features_cnt,
   if (all.permutations)
   {
     // just multiply precomputed values for all namespaces
-    for (std::vector<uint8_t>& inter : *ec.interactions)
+    for (auto inter : *ec.interactions)
```

Comment:
This is now a copy on each iteration, use a ref

Reply:
Updated